src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/model/EditableAdjustmentDto.kt

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.digital.hmpps.adjustments.api.model
+
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.adjustments.api.entity.AdjustmentType
+import java.time.LocalDate
+import java.util.UUID
+
+@Schema(description = "Create/ edit an adjustment")
+data class EditableAdjustmentDto(
+  @Schema(description = "The ID of the adjustment")
+  val id: UUID?,
+  @Schema(description = "The NOMIS booking ID of the adjustment")
+  val bookingId: Long,
+  @Schema(description = "The NOMIS ID of the person this adjustment applies to")
+  val person: String,
+  @Schema(description = "The type of adjustment")
+  val adjustmentType: AdjustmentType,
+  @Schema(description = "The end date of the adjustment")
+  val toDate: LocalDate?,
+  @Schema(description = "The start date of the adjustment")
+  val fromDate: LocalDate?,
+  @Schema(description = "The number of adjustment days")
+  val days: Int?,
+  @Schema(description = "The details of a remand adjustment")
+  val remand: RemandDto?,
+  @Schema(description = "The details of an additional days awarded adjustments (ADA)")
+  val additionalDaysAwarded: AdditionalDaysAwardedDto?,
+  @Schema(description = "Additional details of a UAL adjustment")
+  val unlawfullyAtLarge: UnlawfullyAtLargeDto?,
+  @Schema(description = "The details of a tagged-bail adjustment")
+  val taggedBail: TaggedBailDto?,
+
+  @Schema(description = "The prison where the prisoner was located at the time the adjustment was created (a 3 character code identifying the prison)", example = "LDS")
+  val prisonId: String? = null,
```

Comment:
I don't think the prisonId can actually be set via the API, so I'd say it wasn't edittable.

Reply:
Its being set in the create (https://github.com/ministryofjustice/hmpps-adjustments-api/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/service/AdjustmentsService.kt#L76) and update (https://github.com/ministryofjustice/hmpps-adjustments-api/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/service/AdjustmentsService.kt#L195) methods

Reply:
Oh yea, no idea how I missed that. Maybe it shouldn't be nullable then!